### DISCUSSION TITLE
Support for Folia servers

### DISCUSSION THREAD

**Discussion by ricsal1** [Lorenzo0111/QualityArmory#569] [General]
Hello, I have made quite some changes to support Folia servers.
Are you interested on having this support?

**Comment by Lorenzo0111** [comment]
If you can open a pull request, i'll look into it!

**Reply by ricsal1** [reply]
Lo m8, I don't have permission, it doesn't accept my user for the submission
![imagem](https://github.com/Lorenzo0111/QualityArmory/assets/25226624/58d047ab-10ae-4728-8b5e-3ca205c77cde)

**Reply by Lorenzo0111** [reply]
The new repository url is https://github.com/Lorenzo0111/QualityArmory

**Reply by ricsal1** [reply]
Still having problems, I must be doing something wrong with this

**Reply by ricsal1** [reply]
forget, I need to remake everything on this repository :/

**Reply by Lorenzo0111** [reply]
try making it from the website

**Reply by ricsal1** [reply]
When i try to push the folia branch to git

![imagem](https://github.com/Lorenzo0111/QualityArmory/assets/25226624/47190bc7-a8b1-4145-9f60-7b54846c2300)

**Reply by Lorenzo0111** [reply]
> When i try to push the folia branch to git
> 
> 
> 
> ![imagem](https://github.com/Lorenzo0111/QualityArmory/assets/25226624/47190bc7-a8b1-4145-9f60-7b54846c2300)
> 
> 

you need to push it into a fork that you make, then you can pr
